src/asciidoc/docs/distribute/app-store.adoc

```diff
@@ -0,0 +1,94 @@
+= Listing in the App Store
+:page-slug: app-store
+:page-description: An introduction to the Koji App Store and how to list your Koji app.
+:includespath: ../_includes
+
+After you create, test, and publish your app, you can share its production URL or list the app in the Koji App Store.
+
+The https://withkoji.com/apps[Koji App Store] is the marketplace for all Koji apps.
+When creators edit their Koji profile, they can choose to add an app from the App Store.
+
+By listing in the Koji App Store, you can take advantage of the traffic in the App Store to get exposure for your app.
+Koji also announces noteworthy apps in social media posts.
+
+This topic shows how to create a Koji App Store listing after you move your app to production.
+
+NOTE: You can have multiple listings for the same app.
```

Comment:
While this is technically true, does it potentially create confusion to make a note of it without explaining why one might do this?

Reply:
Removing the note for now. Come to think of it, this might actually open up a can of worms.